testsuite/features/qam/init_clients/branch.feature

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped if there is no proxy
```

Comment:
I see we have the same comment in the feature files at the core.
Probably would be OK to drop those header comments: all the tags do exactly that, neither `@proxy` nor `@private_net` is special.

Reply:
Good point. I will take a look at this. (Maybe we can change it all at once then.)